src/Module.h

```diff
@@ -277,6 +277,20 @@ class Module {
     */
     int getTx() const { return(_tx); }
 
+    /*!
+      \brief Access method to get DIO2 RF switch flag.
+
+      \returns true if DIO2 is set as RF switch.
+    */
+    int getDio2Func() const { return(_dio2RfSwitch); }
+
+    /*!
+      \brief Access method to set/reset DIO2 RF switch flag.
+
+      \returns true if DIO2 is set as RF switch.
+    */
+    void setDio2Func(bool enable);
+
```

Comment:
Module class should not contain module-specific code.

Reply:
So it should be in SX126x.h and SX126x.cpp ?

Reply:
Do we need these methods at all? We can keep the DIO2 state as a variable in SX126x class.